stubs/google/cloud/tasks_v2/__init__.pyi

```diff
@@ -0,0 +1,30 @@
+from google import auth
+from google.api_core.retry import Retry
+from google.cloud.tasks_v2 import types
+
+from typing import Any, Dict, Optional, Sequence, Text, Tuple, Union
+
+
+class CloudTasksClient(object):
+    def __init__(
+            self,
+            credentials: auth.credentials.Credentials = ...
+    ) -> None: ...
+
+    @classmethod
+    def queue_path(cls, project: Text, location: Text, queue: Text) -> Text: ...
+
+    def create_task(
+        self,
+        parent: Text,
+        task: Union[types.Task, Dict[Text, Any]],
```

Comment:
Can we get rid of this `Any` here?

Reply:
Please take a look at the [doc](https://docs.google.com/document/d/1LEWKK2H303eHmWg1hHN2EMIlnewB6EnEyrQoReY-M6s/edit?disco=AAAAN7YgTKY). It explains why task has to be of type Any. Thanks!